cmd/server.go

```diff
@@ -80,5 +82,6 @@ var serverCmd = &cobra.Command{
 }
 
 func init() {
+	serverCmd.PersistentFlags().StringVar(&port, "port", "50051", "server port (default is 50051")
```

Comment:
Will this conflict with #20 setting the default port to 9090?

Reply:
yeah, i'll need to update this. I'd prefer to come back around and do that as this PR is already large and moving quite slow.